apps/src/templates/header-banner.module.scss

```diff
@@ -0,0 +1,72 @@
+@import "breakpoints";
+@import "typography";
+
+.banner {
+  // background-image is passed in via the `style` attribute
+  background-size: cover;
+  position: relative;
+  padding: 2rem 1rem;
+  box-sizing: border-box;
+
+  /*
+    The banner background image leaves an open space for text on the left. Therefore, in Right-to-Left languages, the
+    banner needs to be flipped horizontally so the background image empty space is on the right.
+   */
+  html[dir="rtl"] & {
+    transform: scaleX(-1);
+  }
```

Comment:
TIL! 😮

Reply:
YEAH, this was wild to learn about. And then having to "undo" the reversal on the text ... 🤯

Reply:
I saved this in my Codebase Notes doc for future banners!